npm/translatePolicy.go

```diff
@@ -232,7 +233,7 @@ func translateIngress(ns string, policyName string, targetSelector metav1.LabelS
 		if portRuleExists && !fromRuleExists && !allowExternal {
 			for _, portRule := range rule.Ports {
 				if portRule.Port != nil && portRule.Port.IntValue() == 0 {
-					portName := portRule.Port.String()
+					portName := "port_" + portRule.Port.String()
```

Comment:
Not all Port rules have named ports. can we confirm before adding prefix to all port rules

Reply:
>portRule.Port.IntValue() == 0

This check above is identifying that port has no int value and is alphanumeric. This suggests we do not support numerical name for namedPorts.